app/commands/exercise/export_solutions_to_zip_file.rb

```diff
@@ -0,0 +1,28 @@
+require 'zip'
+
+class Exercise::ExportSolutionsToZipFile
+  include Mandate
+
+  initialize_with :exercise
+
+  def call
+    file_stream = Zip::OutputStream.write_buffer do |zip|
+      solutions.each.with_index do |solution, idx|
+        solution.latest_iteration.files.each do |iteration_file|
```

Comment:
I've used the v2 version of the code as inspiration, but I was wondering whether we might want to output the files like the representer/analyzer would receive them (which would include test files and such).

Reply:
With a maintainer hat on, I'd probably rather just get the students' files as those are the ones that I have to deal with. But if your maintainers hat (or other people) disagree, I don't have any strong preference. Do we actually pass test files etc to representers?

Reply:
> Do we actually pass test files etc to representers?

We do. All exercise files are passed to the tooling, except for the files within `.docs` (and the solution files overwrite any exercise files).

> But if your maintainers hat (or other people) disagree,

The reason why we do the above, is that some tracks might have non-solution files (e.g. project files) that are needed for the representer/analyzer to work well. E.g. the F# analyzer might want to analyze the project file for the order of file dependencies, which is not a solution file.